avocado/core/nrunner.py

```diff
@@ -311,7 +311,28 @@ def run(self):
 
 
 class TAPRunner(BaseRunner):
-    """Runner for standalone executables treated as TAP."""
+    """Runner for standalone executables treated as TAP
```

Comment:
For history sake, this should be squashed into the (previous) commit that introduces the runner.

Reply:
Fixed on v3.